README.md

```diff
@@ -89,6 +89,53 @@ Basic proof of 'ipfs working' locally:
 	ipfs cat <that hash>
 
 
+### Docker usage
+
+An ipfs docker image is hosted at [hub.docker.com/u/ipfs/go-ipfs](hub.docker.com/u/ipfs/go-ipfs).
```

Comment:
@mlovci ah sorry, didnt mean to change it to this yet -- I dont have control of that account atm (only that i _want_ to use it). will update when i get control

i did update https://registry.hub.docker.com/u/jbenet/go-ipfs/ to point to https://github.com/ipfs/go-ipfs/ (instead of https://github.com/jbenet/go-ipfs/)